src/librustc_codegen_llvm/type_.rs

```diff
@@ -233,11 +251,26 @@ impl Type {
         }
     }
 
-    pub fn ptr_to(&self) -> &Type {
+    pub fn ptr_to(&'ll self) -> &'ll Type {
+        self.ptr_to_as(AddrSpaceIdx(0))
```

Comment:
I'm asking because you hardcoded `0` here.

Reply:
This isn't, specifically, creating a `flat` pointer. It's creating a pointer in the default space.

Reply:
I see. I hope we can, at least for now, require that they're one and the same.